res/TensorFlowLiteRecipes/Inf_Pad_000/test.rule

```diff
@@ -0,0 +1,6 @@
+# To check if dynamic dimension properly inferred
+
+RULE    "VERIFY_FILE_FORMAT"      $(verify_file_format) '=' 1
+
+RULE    "PAD_EXIST"               $(op_count PAD) '=' 1
```

Comment:
IMHO, `PAD_EXIST` is not related to SHAPE_INFERENCE.
Why is this necessary?

Reply:
> Why is this necessary?

Your right.